db/migrate/20190604172236_create_widgets.rb

```diff
@@ -5,6 +5,7 @@ def change
       t.boolean :enabled, null: false
       t.integer :duration_seconds, null: false
       t.integer :position, null: false
+      t.references :location
```

Comment:
probably we should not be changing an existing migration like this, better to create a new migration instead